sentry-apollo-3/src/main/java/io/sentry/apollo3/SentryApollo3HttpInterceptor.kt

```diff
@@ -108,87 +164,238 @@ class SentryApollo3HttpInterceptor @JvmOverloads constructor(private val hub: IH
             urlDetails.applyToSpan(this)
 
             operationId?.let {
-                setData("operationId", it)
+                setData("operation_id", it)
             }
 
             variables?.let {
+                // TODO: this is PII but it is here, what do we do?
                 setData("variables", it)
```

Comment:
@lbloder @adinauer do you recall if there was a conversation around PII?

Reply:
Yeah, there was some discussion in the initial implementation of apollo 3 (https://github.com/getsentry/sentry-java/pull/2109#discussion_r911027397). At the time we decided against guarding the variables with defaultPII

Reply:
I will keep it then.